src/cdk/layout/breakpoints-observer.ts

```diff
@@ -75,16 +75,16 @@ export class BreakpointObserver implements OnDestroy {
 
     const mql: MediaQueryList = this.mediaMatcher.matchMedia(query);
     // Create callback for match changes and add it is as a listener.
-    const queryObservable = fromEventPattern(
+    const queryObservable = fromEventPattern<MediaQueryList>(
       // Listener callback methods are wrapped to be placed back in ngZone. Callbacks must be placed
       // back into the zone because matchMedia is only included in Zone.js by loading the
       // webapis-media-query.js file alongside the zone.js file.  Additionally, some browsers do not
       // have MediaQueryList inherit from EventTarget, which causes inconsistencies in how Zone.js
       // patches it.
-      (listener: MediaQueryListListener) => {
+      (listener: Function) => {
```

Comment:
Is there no type more specific than `Function` that we can use here?

Reply:
I don't think so. The `strictFunctionTypes` seem to resolve it to `Function`. A workaround it is to have `as MediaQueryListener` before calling, but that seemed a little ugly.

Reply:
Aright, sg